### PR TITLE
[stable/memcached] Add updateStrategy configuration option

### PR DESCRIPTION
**Pull request by RemingtonReackhof**
#### What this PR does / why we need it:
Adds updateStrategy as a configurable value to the memcached chart. Some memcached deployments/statefulsets are sensitive and cannot be rolled out after each update. Adding the option to choose when to update memcached adds flexibility to the rollout process. 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/memcached/templates/statefulset.yaml [hunk 1]
```diff
@@ -119,3 +119,7 @@ spec:
       affinity:
 {{ toYaml . | trim | indent 8}}
 {{- end }}
+{{- with .Values.updateStrategy }}
```
Comment (olemarkus) [inline stable/memcached/templates/statefulset.yaml]: Looks like other charts have opted for setting specific keys here. Like `updateStrategy.type`. Should probably do the same here

**Review by olemarkus — COMMENTED** [review]
Thanks for the contribution. See the inline comment

File: stable/memcached/templates/statefulset.yaml (hunk 1 above)
Comment (RemingtonReackhof) [reply stable/memcached/templates/statefulset.yaml]: Gotcha, thanks for the suggestion

File: stable/memcached/README.md [hunk 2]
```diff
@@ -62,6 +62,7 @@ The following table lists the configurable parameters of the Memcached chart and
 | `securityContext.enabled`  | Enable security context    | `true`                                                       |
 | `securityContext.fsGroup`  | Group ID for the container | `1001`                                                       |
 | `securityContext.runAsUser`| User ID for the container  | `1001`                                                       |
+| `updateStrategy`           | Update strategy for the StatefulSet/Deployment | `RollingUpdate`                          |
```
Comment (olemarkus) [inline stable/memcached/README.md:66]: I suppose this part should be changed too

**Comment by RemingtonReackhof** [comment]
@olemarkus I made that adjustment, waiting for any further feedback. Thanks!

**Pull request merged**